items/bows/Widowhail.yml

```diff
@@ -0,0 +1,5 @@
+acquisitionMethods:
+  - type: vendorRecipe
```

Comment:
I'm not sure we should put it as a vendor recipe because you don't vendor the items to get the outcome, but it looks like a good use of the `plainText` method to me. WDYT?

Reply:
yes, I had it as plaintext as first but I changed it to vendor recipe :) ill rewrite it to plaintext then :)

Reply:
Sounds good! So please feel free to push another commit and I'll merge right away :)

Reply:
Hi @manweCZ, hope all is well! Just wanted to check if you had a chance to do this yet? (perhaps committed but not pushed)

Reply:
I just pushed the update.

I had the repo cloned in my work and not at home.